operator/webhooks/pod_mutating_webhook.go

```diff
@@ -60,6 +61,18 @@ func (a *PodMutatingWebhook) Handle(ctx context.Context, req admission.Request)
 		return admission.Errored(http.StatusBadRequest, err)
 	}
 
+	// check if Lifecycle Controller is enabled for this namespace
+	namespace := &corev1.Namespace{}
+	if err = a.Client.Get(ctx, types.NamespacedName{Name: req.Namespace}, namespace); err != nil {
+		logger.Info("could not get namespace", "namespace", req.Namespace)
+		return admission.Errored(http.StatusInternalServerError, err)
```

Comment:
I would not block a manifest to be applied if the operator has problems reaching to the Kube API

Reply:
To be honest, it should fail in this case (we don't know if it's good to apply or not), and the controller which generates the pod will try it again in any way ...